dashboard/app/coverage_batch.go

```diff
@@ -50,22 +50,28 @@ func handleBatchCoverage(w http.ResponseWriter, r *http.Request) {
 			log.Infof(ctx, "there is no new coverage for merging available in %s", ns)
 			continue
 		}
+		nsCovConfig := nsConfig.Coverage
 		if err := createScriptJob(
 			ctx,
-			nsConfig.Coverage.BatchProject,
-			nsConfig.Coverage.BatchServiceAccount,
-			batchScript(ns, repo, branch, 7, dates),
-			nsConfig.Coverage.BatchScopes); err != nil {
-			log.Errorf(ctx, "failed to batchScript(): %s", err.Error())
+			nsCovConfig.BatchProject,
+			nsCovConfig.BatchServiceAccount,
+			batchScript(ns, repo, branch, 7, dates, nsCovConfig.jobInitScript, nsCovConfig.syzEnvInitScript),
+			nsCovConfig.BatchScopes); err != nil {
+			log.Errorf(ctx, "failed to batchScript: %s", err.Error())
 		}
 	}
 }
 
-func batchScript(ns, repo, branch string, days int, datesTo []civil.Date) string {
-	script := "git clone --depth 1 --branch master --single-branch https://github.com/google/syzkaller\n" +
+func batchScript(ns, repo, branch string, days int, datesTo []civil.Date,
+	jobInitScript, syzEnvInitScript string) string {
+	script := jobInitScript
```

Comment:
Don't you need "; " at the end if this script is non-empty?

Reply:
Good question. Let's add "\n" here.

Reply:
Updated. PTAL.